quickstarts/microsoft.databricks/databricks-all-in-one-template-for-vnet-injection-privateaccess/azuredeploy.json

```diff
@@ -0,0 +1,292 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "metadata": {
```

Comment:
remove this or add the bicep file to the sample

Reply:
Sorry for the delay, here. will try to incorporate the review comments at the earliest.

Reply:
>Thanks, we have removed the Metadata block and will commit.